.rubocop_todo.yml

```diff
@@ -29,15 +35,18 @@ Lint/UselessAssignment:
     - 'app/models/supplejack_api/support/concept/searchable.rb'
     - 'app/models/supplejack_api/user.rb'
 
-# Offense count: 70
+# Offense count: 64
+# Configuration parameters: IgnoredMethods, CountRepeatedAttributes.
 Metrics/AbcSize:
-  Max: 51
+  Max: 54
```

Comment:
Not very keen to have those values increased as some of that code is already complicated. Don't know why it was updated though as the code changes are pretty minimal.

Reply:
Fair enough! This has come because I ran rubocop -a and then reran the rubocop todo generator.